### PR TITLE
new cis issue for Digital Insurance Passport

### PR DESCRIPTION
**Pull request by yael-goede**
made a new issue with a config required for the new card as requested by achmea and verified by Peter from SIDN/IRMA.

### CODE REVIEW

File: cis/Issues/complianceCheck/description.xml [hunk 1]
```diff
@@ -1,117 +0,0 @@
...
-				<en>Assurance level</en>
-				<nl>Betrouwbaarheidsniveau</nl>
-			</Name>
-			<Description>
-				<en>The assurance level with which your identity was verified</en>
-				<nl>Het betrouwbaarheidsniveau waarmee je identiteit was vastgesteld</nl>
-			</Description>
-		</Attribute>
-	</Attributes>
-</IssueSpecification>
```
Comment (ivard) [inline cis/Issues/complianceCheck/description.xml:117]: We don't support that credential specification are being deleted. Apps rely on this configuration and when it's missing they will start complaining.

File: cis/Issues/digitalInsurancePassport/description.xml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+	<ShouldBeSingleton>true</ShouldBeSingleton>
+	<IssueURL>
+		<en>https://proddfppfademo.azurewebsites.net/</en>
+		<nl>https://proddfppfademo.azurewebsites.net/</nl>
+	</IssueURL>
+
+	<!-- <ForegroundColor>default</ForegroundColor>
+	<BackgroundGradientStart>default</BackgroundGradientStart>
+	<BackgroundGradientEnd>default</BackgroundGradientEnd> -->
+	<IsInCredentialStore>true</IsInCredentialStore>
```
Comment (ivard) [inline cis/Issues/digitalInsurancePassport/description.xml:26]: We don't allow demo credentials to be part of the credential overview in the IRMA-app.

File: cis/Issues/complianceCheck/description.xml (hunk 1 above)
Comment (yael-goede) [reply cis/Issues/complianceCheck/description.xml]: I added back the old credential.

File: cis/Issues/digitalInsurancePassport/description.xml (hunk 2 above)
Comment (yael-goede) [reply cis/Issues/digitalInsurancePassport/description.xml]: I disabled the feature by setting the setting to false.

**Review by ivard — DISMISSED** [review]

File: cis/Issues/digitalInsurancePassport/description.xml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	</Name>
+	<ShortName>
+		<en>Demo Digital Insurance Passport</en>
+		<nl>Demo Digitaal Paspoort Verzekeren</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>cis</IssuerID>
+	<CredentialID>digitalInsurancePassport</CredentialID>
+	<Description>
+		<en>With your Digital Insurance Passport, insurers can see who you are and determine you do not appear in national and international sanctions, investigation and incident registers. This check is mandatory by law and insurers do these together to prevent costs and extra work for customers and insurers.</en>
```
Comment (sanderhollaar) [inline cis/Issues/digitalInsurancePassport/description.xml:14]: determine *if* you
but maybe use: and verify that you

do these together -> do these collectively  (although maybe just leave it at 'This check is mandatory by law')

File: cis/Issues/digitalInsurancePassport/description.xml [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+		<nl>Met jouw Digitaal Paspoort Verzekeren kunnen verzekeraars zien wie je bent en kunnen zo vaststellen dat je niet voorkomt in nationale en internationale sanctie-, opsporings- en incidentregisters. Deze controle is wettelijk verplicht en doen verzekeraars samen om kosten en extra werk te voorkomen voor klanten en verzekeraars.</nl>
```
Comment (sanderhollaar) [inline cis/Issues/digitalInsurancePassport/description.xml:15]: ~en kunnen zo~
Misschien 'en doen verzekeraars samen om kosten en extra werk te voorkomen voor klanten en verzekeraars' weglaten

File: cis/Issues/digitalInsurancePassport/description.xml [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+	<FAQContent>
+		<en>We screen your name, date of birth and nationality against national and international sanctions, investigation and incident registers and do not use this data for other purposes. \n• Initials,\n• Prefix,\n• Surname,\n• Gender,\n• Date of birth,\n• Country of birth,\n• Assurance level</en>
+		<nl>We screenen je naam, geboortedatum en nationaliteit tegen nationale en internationale sanctie-, opsporings- en incidentregisters en gebruiken deze gegevens niet voor andere doeleinden. \n• Voorletters,\n• Voorvoegsel,\n• Achternaam,\n• Geslacht,\n• Geboortedatum,\n• Geboorteland,\n• Betrouwbaarheidsniveau</nl>
+	</FAQContent>
+	<FAQHowto>
+		<en>You can load your Digital Insurance Passport in your IRMA app if you follow the steps provided. You will then receive a separate card in your IRMA app with your data and the results of the screening against national and international sanctions, investigation and incident registers. Normally that outcome is negative and you receive the card in the app. In the unlikely event that an extra check is necessary, you will be notified and later informed about whether or not the Digital Insurance Passport has been granted.</en>
```
Comment (sanderhollaar) [inline cis/Issues/digitalInsurancePassport/description.xml:44]: You can ~load~ add
~in~to your IRMA app

File: cis/Issues/digitalInsurancePassport/description.xml [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+		<nl>Je kunt je Digitaal Paspoort Verzekeren in je IRMA app ladenals je de gevraagde stappen doorlopt. Je ontvangt daarna een apart kaartje in je IRMA app met je gegevens en de uitkomsten van de screening tegen nationale en internationale sanctie-, opsporings- en incidentregisters. Normaal gesproken is die uitkomst negatief en ontvang je het kaartje in de app. Mocht er onverhoopt een extra controle nodig zijn dan krijg je daarover een melding en later bericht over het al dan niet toekennen van het Digitaal Paspoort Verzekeren. </nl>
```
Comment (sanderhollaar) [inline cis/Issues/digitalInsurancePassport/description.xml:45]: Je kunt je Digitaal Paspoort Verzekeren ~in~ aan je IRMA app ~ladenals~ toevoegen ~als je~ met het doorlopen van de gevraagde stappen ~doorlopt~

Comment (sanderhollaar) [inline cis/Issues/digitalInsurancePassport/description.xml:45]: Normaal gesproken is die uitkomst ~negatief~ positief  (maar die hele zin kan ook weggelaten worden)

File: cis/Issues/digitalInsurancePassport/description.xml [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+			</Description>
+		</Attribute>
+		<Attribute id="assuranceLevel" displayIndex="7">
+			<Name>
+				<en>Assurance level</en>
+				<nl>Betrouwbaarheidsniveau</nl>
+			</Name>
+			<Description>
+				<en>The assurance level with which your identity was verified</en>
+				<nl>Het betrouwbaarheidsniveau waarmee je identiteit is vastgesteld</nl>
```
Comment (sanderhollaar) [inline cis/Issues/digitalInsurancePassport/description.xml:126]: was/is

File: cis/Issues/digitalInsurancePassport/description.xml (hunk 3 above)
Comment (yael-goede) [reply cis/Issues/digitalInsurancePassport/description.xml:14]: nice, missed that one

**Review by ivard — APPROVED** [review]

**Pull request merged**